src/legend_detector_to_ssd.jl

```diff
@@ -63,7 +63,7 @@ function LEGEND_SolidStateDetector(::Type{T}, meta::PropDict, env::Environment,
     # "old" semiconductor 
     sc = detector.semiconductor
     # new semiconductor with our temperature
-    semiconductor = SolidStateDetectors.Semiconductor(temperature, sc.material, sc.impurity_density_model, sc.charge_drift_model, sc.geometry)
+    semiconductor = SolidStateDetectors.Semiconductor(temperature, sc.material, sc.impurity_density_model, sc.charge_drift_model, sc.charge_trapping_model, sc.geometry)
```

Comment:
This change requires the change to `SolidStateDetectors@0.10.3`, that's why we need to update the `SolidStateDetectors` entry in the `Project.toml` to `0.10.3` :+1: